release_notes.md

```diff
@@ -6,6 +6,16 @@ The file describes all the changes, that are affected the usage of the applicati
 
 The format is based on [Release_notes format](https://github.com/ios-course/ironfoudation-team-project/wiki/Release_notes-format).
 
+## 0.3.0
+
+### New features
+
+`Log in`: Add possibility for user to select how to run the app - as a pet owner or a pet sitter.
+
+`Logout`: The user are able to change the selected role (owner or sitter). The  Logout button is displayed in the navigation bar.
+
+`Copy info`: Users are able to copy the name and the phone number of a pet sitter by a long tapping on the sitter's full name and phone information.
+
```

Comment:
I think you need to add another empty line here to be consistent.

Reply:
There are two blank lines between the first and second release.

Reply:
Fixed it. Thanks.